var/spack/repos/builtin/packages/gawk/package.py

```diff
@@ -43,3 +43,11 @@ class Gawk(AutotoolsPackage):
     url      = "http://ftp.gnu.org/gnu/gawk/gawk-4.1.4.tar.xz"
 
     version('4.1.4', '4e7dbc81163e60fd4f0b52496e7542c9')
+
+    depends_on('gettext')
+    depends_on('libsigsegv')
+    depends_on('readline')
+    depends_on('mpfr')
+    depends_on('gmp')
+
+    build_directory = 'spack-build'
```

Comment:
@alalazo This `build_directory` change was necessary to get `make check` to pass for gawk. If I revert it to `Package` instead of `AutotoolsPackage` it also works.

We've seen a lot of packages now that work fine with `./configure` but crash when you run `/absolute/path/to/configure`. I know we added the absolute path logic to allow configure scripts in different directories and building out of source, but I wonder if we should be using `os.path.relpath` instead.